### ISSUE TITLE
one-to-many relationships in Rails

### ISSUE THREAD

**Issue by anhpnguyen** [anhpnguyen/tc359_rails_final#7]
@chrisvfritz Hi Chris! I created a scaffold named "author". Basically a story will have many author ( or contributors). The idea is when a user creates a new story, that story can have many authors who are able to add new contents to that story (so we can have a completed story in the end that is written by many authors). 
My problem is I'm not sure where to start after creating the model "author". I tried some tutorials but since my authentication isn't working, I can't test it. I've been trying to figure it out (submit button doesn't do anything). I thought it was the class of the submit button which I changed from 'form' to 'form-actions' but it's still not working. Can you give me some inputs? Thanks in advance.

**Comment by chrisvfritz** [comment]
It looks like your authentication isn't working because none of your inputs are actually inside of your form, but next to it. You'll need to [indent these lines](https://github.com/anhpnguyen/tc359_rails_final/blob/master/app/views/sessions/new.html.slim#L3-L12) so that Slim renders them as inside the form element. Does that make sense?

**Comment by anhpnguyen** [comment]
@chrisvfritz Yes! I fixed it! I start to get the hang of Slim now! I will continue to work on this until I have more questions. Thanks Chris!

**Comment by chrisvfritz** [comment]
No problem :smiley:
